plugins/plugin-optimize/test/stubs/minimal/style.css

```diff
@@ -1 +1,4 @@
-body{border:1px solid red}
\ No newline at end of file
+body {
+  border: 1px solid #ffffff;
+  margin: 1px 1px 1px 1px;
```

Comment:
These changes were for explicitly css minify:
`body{border:1px solid #fff;margin:1px}`